event_model/__init__.py

```diff
@@ -71,8 +75,28 @@ def __call__(self, name, doc, validate=False):
             The same name as what was passed in, and a doc that may be the same
             instance as doc, a copy of doc, or a different dict altogether.
         """
+        if name == 'start':
+            if self._start_doc is None:
+                self._start_doc = doc
+            elif self._start_doc['uid'] == doc['uid']:
+                warnings.warn(f'DocumentRouter received the same start document twice: uid {doc["uid"]}')
```

Comment:
maybe not process it the second time?

Reply:
That was my first thought but the tests for Filler fail so I punted.